rudder-agent/SOURCES/Makefile

```diff
@@ -570,6 +588,12 @@ build-curl: curl-source
 	cd curl-source && $(MAKE) install DESTDIR=$(CURDIR)/dependencies
 	touch $@
 
+build-jq: jq-source
+	cd jq-source && autoreconf -fi && ./configure --prefix=$(RUDDER_DIR) --enable-gettext=no $(BUILD_FLAGS) --disable-maintainer-mode --disable-docs --with-oniguruma=builtin
```

Comment:
what does --enable-gettext=no do ?

Reply:
Oh I missed it, should not be there, bad copy/pasting...